pkg/vm/engine/tae/tables/block.go

```diff
@@ -749,6 +751,7 @@ func (blk *dataBlock) onCheckConflictAndDedup(rowmask *roaring.Bitmap, ts types.
 			blk.mvcc.RLock()
 		}
 		if deleteNode.IsAborted() || !deleteNode.IsVisible(ts) {
+			logutil.Infof("table %d-%v,row %v pk %v", blk.meta.GetSegment().GetTable().ID, blk.meta.GetSchema().Name, row, blk.pkIndex)
```

Comment:
You can comment on these print by default and enable these print when debugging. It's too noisy.

Reply:
So how should I debug when there is a bug?

Reply:
Use debug level